src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+
+    public final String value;
```

Comment:
Is it really alright to leave this as public? Perhaps we can look into the `protected` scope to achieve the same intended functionality? =P